src/events/guild/member/guildMemberAdd.ts

```diff
@@ -29,9 +31,25 @@ export default class extends Event {
 
 		if (member.pending) return;
 
-		const config = await databaseManager.getConfig(member.guild.id)
+		const mute: Record<string, any> = await getLatestMute(member, member.guild.id);
+		if (mute) {
+			const muteRole: Snowflake = <Snowflake>await databaseManager.getMuteRole(<Snowflake>member.guild?.id);
+			if (!muteRole) return;
```

Comment:
```suggestion
			const muteRole: Snowflake | null = await databaseManager.getMuteRole(<Snowflake>member.guild?.id);
			if (!muteRole) return;
```
We should avoid typecasting when possible since it can mask and/or cause bugs. Since we need to check for the null anyway we might as well declare it. After the null check the type narrows to only `Snowflake` so it'll behave the same.